packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -46,6 +46,30 @@ async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
     );
 }
 
+async function growTableToGivenRowWidth(
+    rowWidth: number,
+    table: Table<SimpleTableRecord>
+): Promise<void> {
+    if (!table.$fastController.isConnected) {
+        throw Error(
+            "You must call 'await connect()' before calling this method"
```

Comment:
nit: It's not 100% true that you have to call `await connect()` before calling this. Instead, can this error say something like "The element must be connected before calling this method"?